WordGame/Scenes/Home/HomeViewController.swift

```diff
@@ -0,0 +1,114 @@
+//
+//  HomeViewController.swift
+//  WordGame
+//
+//  Created by Kumari Ritu Pal on 25/07/22.
+//
+
+import UIKit
+
+class HomeViewController: UIViewController, StoryboardHelper {
+    var interactor: HomeBuisnessLogic?
+    var viewModel: HomeViewModel?
+    
+    //MARK:- IBOutlets
+    @IBOutlet weak var lblCorrectAttempts: UILabel!
+    @IBOutlet weak var lblWrongAttempts: UILabel!
+    @IBOutlet weak var lblSpanishWord: UILabel!
+    @IBOutlet weak var lblEnglishWord: UILabel!
+    @IBOutlet weak var BtnCorrect: UIButton!
+    @IBOutlet weak var BtnWrong: UIButton!
+    
+    //MARK:- View Life Cycle
+    override func viewDidLoad() {
+        super.viewDidLoad()
+//        self.defaultSetup()
+        self.setupView()
+        self.getLanguageWord()
+    }
+    
+    //MARK:- Methods
+    func defaultSetup() {
```

Comment:
```defaultSetup()``` is not being called anywhere.
Its commented out on line 25. So it seems like dead code.